src/types/plugin-input.ts

```diff
@@ -23,12 +25,20 @@ export const startStopSchema = T.Object({
   ),
   miscellaneous: T.Object(
     {
-      maxConcurrentTasks: T.Number(),
-      startRequiresWallet: T.Boolean(),
+      startRequiresWallet: T.Boolean({ default: true }),
+      maxConcurrentTasks: T.Record(T.String(), T.Integer(), { default: { admin: 20, member: 10, contributor: 2 } }),
     },
-    { default: { maxConcurrentTasks: 3, startRequiresWallet: true } }
+    {
+      default: {},
+        maxConcurrentTasks: {
```

Comment:
This should be removed, this is not event part of the arguments of `Type.Object` proper options.